croncert-config.yml

```diff
@@ -412,6 +412,53 @@ scrapers:
         regex: ".*ABGESAGT.*"
         match: false
 
+  ############
+  # Cully
+  ############
+  - name: CullyJazz
+    url: https://cullyjazz.ch/fr/programme
+    item: body > div > main > div > div > div > section.row > div.event-box-wrapper > div.paid-events > div.event-box > div.event-box-inner > a
+    fields:
+      - name: city
+        value: Cully
+      - name: type
```

Comment:
this has to be 'type' instead of 'evenType' otherwise the event api will reject this.

Reply:
OK. I think I'm confused about the difference between the two. Do you just use eventType for filters? And type is for the API?

Reply:
type is a mandatory field if we want to write to the API which is the case for this repository, see https://github.com/jakopako/event-api/blob/main/models/Event.go#L22 

eventType is an arbitrary name that I sometimes use to filter events. I admit that the naming here is a bit suboptimal.